libraries/wutdevoptab/devoptab_fs_statvfs.c

```diff
@@ -5,7 +5,51 @@ __wut_fs_statvfs(struct _reent *r,
                  const char *path,
                  struct statvfs *buf)
 {
-   //TODO: FSGetFileSystemInfo
-   r->_errno = ENOSYS;
-   return -1;
+   FSStatus status;
+   FSCmdBlock cmd;
+   uint64_t freeSpace;
+   FSStat driveStat;
+   memset(buf, 0, sizeof(struct statvfs));
+
+   char *fixedPath = __wut_fs_fixpath(r, path);
+   if (!fixedPath) {
+      return -1;
+   }
+
+   FSInitCmdBlock(&cmd);
+
+   status = FSGetFreeSpaceSize(__wut_devoptab_fs_client, &cmd, fixedPath, &freeSpace, FS_ERROR_FLAG_ALL);
```

Comment:
This is really slow for sd cards (20+ seconds) =(

Reply:
Reverted the implementation in the latest commit. I don't really think there's a great way to implement it without removing Nintendo's slow fat32 driver. Might be faster on emptier/smaller SD cards, but I don't think it's worth having if it means that homebrew devs might think their homebrew froze due to it taking so long.